assets/armsnippets.jsonc

```diff
@@ -967,7 +966,8 @@
             "                       \"path\": \"/${9:PathToNotIndex}/*\"",
             "                }]",
             "            }",
-            "        }",
+            "        },",

```

Comment:
Can you create a cosmos graph in the portal and compare?  They've made some important changes since GA'ing that likely aren't in here.